[OSDOCS-20987]: Document Agent-based Installer rendezvous host port 8090/TCP (#116975)
openshift/openshift-docs: modules/agent-install-networking.adoc

Do not set the `network.machineNetwork.cidr` parameter to include this address range, otherwise a conflict causes the cluster installation to fail.
====

[id="agent-install-networking-ports_{context}"]
== Port requirements for the rendezvous host

> During the discovery and bootstrap phases of an installation, all non-rendezvous hosts connect to the Assisted Service that runs on the rendezvous host. Configure your firewall to allow the following traffic from each host to the rendezvous host:

Reviewer: I'd suggest to amend the `all non-rendezvous hosts` phrase to `all the hosts`. Technically speaking, the rendezvous host node connects to itself for registering in AS

> During the discovery and bootstrap phases of an Agent-based installation, all non-rendezvous hosts connect to the Assisted Service that runs on the rendezvous host. Configure your firewall to allow the following traffic from each host to the rendezvous host:

Reviewer 2: Similar note about "Agent-based installation"
```suggestion
During the discovery and bootstrap phases of an installation, all non-rendezvous hosts connect to the Assisted Service that runs on the rendezvous host. Configure your firewall to allow the following traffic from each host to the rendezvous host:
```

.Ports required to reach the Assisted Service on the rendezvous host
[cols="2a,2a,5a",options="header"]
|===
|Protocol
|Port
|Description

|TCP

> |`8090`

Reviewer 2: Could you swap the Protocol and Port columns? Makes more sense to me to mention the port first:

```suggestion
|Port
|Protocol
|Description

|`8090`
|TCP
```

|Assisted Service API. Hosts use this port to register with the Assisted Service, report hardware information, and retrieve installation instructions.
|===

[NOTE]
====
Port `8090` is required only during installation. After installation completes, the Assisted Service is no longer exposed on the rendezvous host.
====

> This specific port requirement is in addition to the standard {product-title} networking requirements for installation, such as connectivity to the Machine Config Server on ports `22623` and `22624`.

Reviewer: Also here I'd remove the MCS mention, leaving a link to the std requirements

> This Agent-based Installer-specific requirement is in addition to the standard {product-title} networking requirements for installation, such as connectivity to the Machine Config Server on ports `22623` and `22624`.

Reviewer 2: ```suggestion
This specific port requirement is in addition to the standard {product-title} networking requirements for installation, such as connectivity to the Machine Config Server on ports `22623` and `22624`.
```
